README.md

```diff
@@ -1,28 +1,73 @@
 # terraform-provider-flux
 
-This is a Terraform provider for Flux v2, it enables bootstrap a Kubernetes custer with Flux v2 using terraform.
+This is the Terraform provider for Flux v2.
+The provider allows you to install Flux on Kubernetes
+and configure it to reconcile the cluster state from a Git repository.
 
 ## Example Usage
 
-The provider is consists of two data sources `flux_install` and `flux_sync` the data sources are corresponding to [fluxv2 manifests](https://pkg.go.dev/github.com/fluxcd/flux2@v0.2.1/pkg/manifestgen)
+The provider consists of two data sources `flux_install` and `flux_sync`,
+the data sources are corresponding to [fluxv2 manifests](https://pkg.go.dev/github.com/fluxcd/flux2@v0.2.1/pkg/manifestgen).
 
-The data sources are returing `YAML` manifest so a second provider is needed to apply the manifest into the Kubernetes cluster. See example folder.
-
-The `flux_install` generates manifests to install the `fluxv2` components.
+The `flux_install` data source generates a multi-doc YAML with Kubernetes manifests that can be used to install or upgrade Flux:
 
 ```hcl
-# Flux
+# Generate manifests
 data "flux_install" "main" {
-  target_path = "staging-cluster"
+  target_path    = "production"
+  arch           = "amd64"
+  network_policy = false
+  version        = "latest"
+}
+
+# Load generated manifests
+data "kubectl_file_documents" "apply" {
```

Comment:
We should probably link to the provider we are using.

https://registry.terraform.io/providers/gavinbunney/kubectl/latest